src/dbcp/extract/local_opposition.py

```diff
@@ -91,7 +90,7 @@ def _remove_intro(
                 return paragraphs[idx:]
         raise ValueError("Could not find starting state")
 
-    def _parse_values(self, text: str) -> None:
+    def _parse_values(self, text: str) -> None:  # noqa: C901
```

Comment:
non-blocking: it might be worth pulling each section handler out into its own function, then using `match/case` or a dictionary to handle dispatch based on each section.

Are the sections always in a specific order? Or can they be a random jumble?